### PR TITLE
only build workflows should go in build workflow queue

### PR DESCRIPTION
**Pull request by nnutter**
This doesn't really solve the problem pointed out in RT#104979 but instead
probably just moves it to another queue.  A better solution should be possible
after the LSF queues are redesigned in a couple weeks.

### CODE REVIEW

File: lib/perl/Genome/Process/Command/Run.pm [hunk 1]
```diff
@@ -148,7 +148,7 @@ sub _bsub_in_pend_state {
         log_file => $self->workflow_process_out_log,
         hold_job => 1,
         project => $self->process->workflow_name,
-        queue => $ENV{GENOME_LSF_QUEUE_BUILD_WORKFLOW},
+        queue => $ENV{GENOME_LSF_QUEUE_DV2_WORKFLOW},
```
Comment (tmooney) [inline lib/perl/Genome/Process/Command/Run.pm]: I would make `Genome::Process::Command::Run` an exception to this change.  `Process`es and `Build`s are nearly identical in terms of running workflows (and I still hope they are eventually deduplicated!).

Comment (nnutter) [inline lib/perl/Genome/Process/Command/Run.pm]: Don't Builds run Processes?

Comment (tmooney) [inline lib/perl/Genome/Process/Command/Run.pm]: I believe that a `Build` that runs a `Process` should `execute` it rather than `schedule` it.

Comment (nnutter) [inline lib/perl/Genome/Process/Command/Run.pm]: Alright, I'll revert this instance.

**Pull request merged**